l10n_it_fatturapa_out_rc/models/account_move.py

```diff
@@ -81,42 +81,42 @@ def preventive_checks(self):
                     "select invoices with same supplier"
                 )
             )
+        rc_supplier = rc_suppliers and rc_suppliers[0] or rc_suppliers.browse()
```

Comment:
Ok che era così anche prima, ma che senso avrebbe fare tutti i controlli qui sotto su un record vuoto (`rc_suppliers.browse()`)?
Potresti semplicemente modificare il controllo qui sopra in modo che sollevi eccezione quando `len(rc_suppliers) != 1`, che ne pensi?

Reply:
In effetti non credo sia possibile a quel punto, abbiamo già selezionato le fatture di reverse charge, almeno un supplier dovrebbe esserci. Però un controllo non guasta.